Output TYPE for getting service

FengyunPan opened the pull request:
**What this PR does / why we need it**:
Now service already supported 4 ServiceTypes, ServiceTypes is
friendly to distinguish services, so outputing service type better
when running 'kubectl get service'.

**Release note**:
```release-note
  NONE
```

xiangpengzhao commented on pkg/printers/internalversion/printers_test.go:
```diff
@@ -2309,7 +2309,7 @@ func TestPrintService(t *testing.T) {
 				},
 			},
-			"test2\t10.9.8.7\t<none>\t8888:9999/tcp\t<unknown>\n",
+			"test2\tNodePort\t10.9.8.7\t<nodes>\t8888:9999/tcp\t<unknown>\n",
```
nodes?

FengyunPan replied:
Yes, I have run unit test.

xiangpengzhao replied:
Why is it not `<none>` but `<nodes>`? I don't think `<nodes>` is a right and meaningful output here.

FengyunPan replied:
I see. I update it soon.

xiangpengzhao commented on pkg/printers/internalversion/printers_test.go:
```diff
@@ -2292,14 +2292,14 @@ func TestPrintService(t *testing.T) {
 					ClusterIP: "0.0.0.0",
```
in fact, this is not a valid Cluster IP. since you are here, can you modify it?

FengyunPan replied:
Ok, Done.

dims commented:
@k8s-bot ok to test

Commit:
Output TYPE for getting service

Now service already supported 4 ServiceTypes, ServiceTypes is
friendly to distinguish services, so outputing service type better
when running 'kubectl get service'.

dims commented:
/assign @fabianofranz 
/unassign

fabianofranz commented:
/lgtm
/approve no-issue

FengyunPan commented:
/test pull-kubernetes-e2e-gce-etcd3
